includes/languages/english/html_includes/define_main_page.php

```diff
@@ -1,4 +1,4 @@
-<a href="http://www.zen-cart.com/book"><img src="images/large/e-start-book.gif" alt="get your manual today" title="Have you got yours yet? Join the 1000's of Zen Cart users that have bought the only comprehensive owners manual !" /></a>
+<a href="//zen-cart.com/book"><img src="images/large/e-start-book.gif" alt="get your manual today" title="Have you got yours yet? Join the 1000's of Zen Cart users that have bought the only comprehensive owners manual !" /></a>

```

Comment:
Considering that the destination site permits `https`, this issue is expected to be resolved if `https:` were used rather than the non-assigned protocol.  I've been advised that use of `https:` in the path will not cause a mixed content message even if the page were presented using `http:` as the concern is providing non-secure information on a secure page. Providing secure content on an insecure page is considered good if not some amount of an improvement...